runtime/CMakeLists.txt

```diff
@@ -352,6 +352,19 @@ if(OMR_DDR)
 		${CMAKE_CURRENT_BINARY_DIR}/omr/gc
 		${CMAKE_CURRENT_BINARY_DIR}/omr
 	)
+
+	# Specify the directories which shouldn't be searched for debug info.
+	# Currently this is only used on zos, and has no effect elsewhere
+	set_property(TARGET j9ddr PROPERTY DDR_OBJECT_EXCLUDES
+		bcutil/test/
+		compiler/
+		gc_tests/
+		jilgen/
+		omr/
+		runtimetools/
+		tests/
```

Comment:
Why the trailing slashes?

Reply:
Also this list is *_not_* the same as in `run_omrddrgen.mk.ftl`. This appears to exclude more: compiler and _all_ of omr.

Reply:
Isn't that already explained in the description?

Reply:
You're right, I missed the explanation of the differences.

Reply:
Actually, I don't think the omr portion is handled properly: eclipse/omr#5479 sets `DDR_OBJECT_EXCLUDES` for `omrddr`, not for `j9ddr`.

Reply:
Trailing slashes are there because `DDR_OBJECT_EXCLUDES` is treated as a list of string prefixes rather than a list of directories

Reply:
> Actually, I don't think the omr portion is handled properly: [eclipse/omr#5479](https://github.com/eclipse/omr/pull/5479) sets `DDR_OBJECT_EXCLUDES` for `omrddr`, not for `j9ddr`.

That is intentional. Since `j9ddr` includes `omrddr`, it will inherit any objects included in it.

Reply:
This deserves some comments so future readers will understand that those exclusions are retroactive (omr has already been included) or derived later.